f2/src/ConfirmCancelPage.js

```diff
@@ -0,0 +1,62 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { Trans } from '@lingui/macro'
+import { H1 } from './components/header'
+import { TrackPageViews } from './TrackPageViews'
+import { Layout } from './components/layout'
+import { Stack, Flex } from '@chakra-ui/core'
+import { Route } from 'react-router-dom'
+import { P } from './components/paragraph'
+import { Button } from './components/button'
+import { useStateValue } from './utils/state'
+
+export const ConfirmCancelPage = () => {
+  const [, dispatch] = useStateValue()
+
+  return (
+    <Route
+      render={({ history }) => (
+        <Layout>
+          <TrackPageViews />
+          <Stack spacing={10} shouldWrapChildren>
+            <H1>
+              <Trans id="confirmCancelPage.title" />
+            </H1>
+            <Flex direction="row" align="center" wrap="wrap" mb={10} mt={10}>
+              <P w="100%">
+                <Trans id="confirmCancelPage.summary" />
+              </P>
+
+              <Button
+                onClick={() => {
+                  dispatch({
+                    type: 'deleteFormData',
+                  })
+                  history.push('/cancelled')
+                }}
+                type="submit"
+                w={{ base: '100%', md: 'auto' }}
+                variantColor="red"
+              >
+                <Trans id="confirmCancelPage.cancelButton" />
+              </Button>
+              <Button
+                onClick={() => history.goBack()}
```

Comment:
This is an awesome function to have!

Reply:
I wonder if we can leverage this on the backbutton component